### ISSUE TITLE
Add helm-hook annotations in order to ensure that the certs will only be generated on chart install

### ISSUE THREAD

**Issue by thectrltab** [vsphere-tmm/helm-charts#37]
Add helm-hook annotations in order to ensure that the certs will only be generated on chart install:
- https://github.com/vsphere-tmm/helm-charts/blob/296619a452f71c894a27a1ebaa865171f2fcdf07/charts/vsphere-csi/templates/snapshot-validation-webhook/validatingwebhookconfiguration.yaml
- https://github.com/vsphere-tmm/helm-charts/blob/296619a452f71c894a27a1ebaa865171f2fcdf07/charts/vsphere-csi/templates/webhook/ValidatingWebhookConfiguration.yaml

```yaml
annotations:
    "helm.sh/hook": "pre-install"
```

Helm docs reference: https://helm.sh/docs/topics/charts_hooks/#the-available-hooks

**Comment by ErikLundJensen** [comment]
Could this is done by adding `validdationWebhook.annotations` to values instead of hardcoding the helm hook annotation?

**Comment by thectrltab** [comment]
Yes of course, I think is a better solution to use the values to solve this.
